src/controllers/index.js

```diff
@@ -6,6 +6,7 @@ const articles = require('./articles');
 const doctors = require('./doctors');
 const signUp = require('./signUp');
 const signIn = require('./signIn');
+const doctor = require('./doctor');
```

Comment:
where is this doctor.js controller ??

Reply:
this is in your local machine but I can't find the file in your PR files

Reply:
It exists now :+1: